snomed/README.md

```diff
@@ -0,0 +1,23 @@
+# SNOMED CT
+
+SNOMED CT (Systematized Nomenclature of Medicine - Clinical Terms) is a standardized, multilingual vocabulary of clinical terminology that is used by physicians and other health care providers for the electronic exchange of clinical health information.
+
+## The goal
+
+Extract data from SNOMED that is relevant to our format.
+
+## Download relevant files
+
+Navigate to https://www.nlm.nih.gov/healthit/snomedct/us_edition.html and download the file.
```

Comment:
1. Maybe add a comment on the US edition `We chose to use this edition as it extends over the official international edition  and should be most updated and full`.
2. its not exactly clear which file to do download, maybe a more concrete link to the download, or a name of the file to download?